uiServices/src/testCaseServices_test.go

```diff
@@ -414,4 +445,84 @@ func TestGetTestCaseFileNotFound(t *testing.T) {
 	}
 
 	assert.Equal(t,http.StatusNotFound, w.Code, "Should not return data")
+}
+
+
+
+func TestSuccessfulCaseDelete(t *testing.T) {
+	r := chi.NewRouter()
+	r.Mount("/", GetIndexPage())
+
+	//create file to be deleted
+	reader := strings.NewReader(TestCaseForDeletion)
+	r.HandleFunc("/test-cases", postTestCase)
+
+	os.Remove(os.Getenv("GOPATH") + "/src/github.com/xtracdev/automated-perf-test/uiServices/test/TestCaseService2.xml")
```

Comment:
Why do you need to delete this file here?

Reply:
Because if that file exists when I rerun a test which involves POST, it will fail as you can't create duplicate files. So I delete the file at start of test so no failures occur.

Reply:
Actually I suppose the test should delete the file so I suppose this can be removed

Reply:
The tests should always clean up in the end. It's just good practice.